instrumented-tests/http-server-mock/python/start_mock_server.py

```diff
@@ -77,34 +69,13 @@ def __GET_inspect(self, parameters):
         inspection_info = []
         for request in history.all_requests():
             inspection_info.append({
-                "request_method": request.http_method,
-                "request_path": request.path,
-                "body_inspection_path": "/inspect/{request_id}/body".format( request_id = request.id ),
-                "headers_inspection_path": "/inspect/{request_id}/headers".format( request_id = request.id )
+                "method": request.http_method,
+                "path": request.path,
+                "body": base64.b64encode(request.http_body), # use Base64 to not corrupt the JSON
```

Comment:
> use Base64 to not corrupt the JSON

what does it mean? does ascii corrupt it for example?

Reply:
In this case, we send JSON within a JSON:
```json
{
   "path": "/foo/bar",
   "body": <json>
}
```
By default, Python escapes the `"body"` json, so it becomes: `"\{\"foo\":\"bar"\}\"`. I found it much easier and less error-prone to just encode it as a base-64 ASCII (easy in Swift) instead of un-escaping the `.utf8` JSON `String` data.